certbot-apache/certbot_apache/configurator.py

```diff
@@ -822,6 +810,8 @@ def make_vhost_ssl(self, nonssl_vhost):  # pylint: disable=too-many-locals
         # We know the length is one because of the assertion above
         # Create the Vhost object
         ssl_vhost = self._create_vhost(vh_p)
+        if self.conf("handle-sites"):
+            ssl_vhost.enabled = False
```

Comment:
Yeah I agree this is the only case the vhost can be in an enabled state, but I think it's possible it's already enabled. While it's maybe a bit of a silly example, I was able to get this PR to fail because it tries to enable an already enabled vhost. Starting with the default Apache2 config on Ubuntu 16.04, the changes I made were:
```
sudo a2dissite 000-default.conf
sudo cp /etc/apache2/sites-available/000-default.conf /etc/apache2/sites-enabled
sudo rm /etc/apache2/sites-available/*
```
In the end, your only vhost is a real file in `sites-enabled` which when we run Certbot it creates a vhost in `sites-enabled` which is marked as not enabled.